common/network_constants.cc

```diff
@@ -7,7 +7,7 @@
 
 const char kBraveProxyPattern[] = "https://*.brave.com/*";
 const char kBraveUpdatesExtensionsEndpoint[] =
-    "https://go-updater.brave.com/extensions";
+    "http://18.191.178.76/extensions";
```

Comment:
did you mean to do this change?

Reply:
I believe @simonhong made this change because the component is not yet published to the production component updater. He made a comment in his branch which hasn't survived the rebase to this branch

Reply:
TODO: revert this before merge

Reply:
this is reverted